30412/Moisa Oana  - Miruna/assignment_2(2)/main.c

```diff
@@ -0,0 +1,126 @@
+#include <stdio.h>
+#include<string.h>
+#include<stdlib.h>
+#define MAX_LINE 100
+#define DATA_PATH_IN "../input.dat"
+#define DATA_PATH_OUT "../output.dat"
+typedef struct node{
+    int data;
+    struct node *next;
+}NodeT;
+typedef  struct list{
+    NodeT *first;
+    NodeT *last;
+}ListT;
+ListT M,T;
+int main() {
```

Comment:
Woah, everything is in the main function, that's not ok!

Please break it up into smaller functions